projects/observability/src/shared/components/timeline-card-list/timeline-card-list.component.test.ts

```diff
@@ -10,7 +11,7 @@ describe('Timeline Card List component', () => {
     declareComponent: false,
     component: TimelineCardListComponent,
     imports: [TimelineCardListModule],
-    providers: []
+    providers: [mockProvider(PopoverService)]
```

Comment:
Don't want to make this a shallow test since I need container component to project contents.